src/data.py

```diff
@@ -3,6 +3,7 @@
 
 from requests import get
```

Comment:
i think i'd actually prefer using `requests.get` just because `get` is so ubiquitous that without the prefix it can be confusing, old code i know but worth a second look